index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt("Bienvenido, como te llamas?");
+alert(`Hola, ${nombreUsuario}!`);
```

Comment:
En este caso me parece que seria mejor que no uses interpolazion de strings.
```suggestion
alert("Hola " + nombreUsuario);
```